src/status_im2/common/standard_authentication/password_input/view.cljs

```diff
@@ -29,19 +30,30 @@
 (defn- view-internal
   [{:keys [default-password theme shell?]}]
   (let [{:keys [error processing]} (rf/sub [:profile/login])
+        auth-method                (rf/sub [:auth-method])
```

Comment:
In my pr I was passing this in as a prop 🤔 No strong preferences, we could keep this input purer however and pass everything in as a prop 🤔 https://github.com/status-im/status-mobile/pull/17992/files

Reply:
I'd say unless it's used higher in the tree, mught be better to use the sub directly where it's used. I see very often 'subscription' maps passed as props all over the place, even when a sub is needed in one single child somewhere down the tree. What do you think?

Reply:
For me it's fine either way 👍